CMakeLists.txt

```diff
@@ -11,23 +11,65 @@ find_package(kodiplatform REQUIRED)
 find_package(p8-platform REQUIRED)
 find_package(Rapidxml REQUIRED)
 find_package(ZLIB REQUIRED)
+find_package(OpenSSL REQUIRED)
```

Comment:
@Rechi I used vfs.sftp and vfs.libarchive as a reference but can't see why this fails on nearly every platform. Am I missing something?

Reply:
OpenSSL is missing in depends.

Reply:
On Jenkins?

Reply:
in your PR branch

Reply:
Oh, now I feel silly. Thanks again for your help.